src/Config.js

```diff
@@ -100,6 +148,29 @@ module.exports = class extends ChainedMap {
               .forEach(name => this.plugin(name).merge(value[name]));
           }
 
+          /**
+           * @see ./Plugin
+           * merge an array of plugins
+           */
+          case 'plugins': {
+            if (Array.isArray(value)) {
```

Comment:
I'm still having trouble seeing why we need this to be an array, when we already have an dictionary collection. It's essentially this:

```js
plugin: {
  name: {
    plugin: Plugin,
    args: [args]
  }
}
```

versus something like this I'm assuming:

```js
plugins: [
  { name, plugin: Plugin, args: [args] }
]
```

How is the current plugin interface lacking?